components/raftstore-v2/src/batch/store.rs

```diff
@@ -14,6 +14,7 @@ use crossbeam::channel::{Sender, TrySendError};
 use engine_traits::{Engines, KvEngine, RaftEngine, TabletFactory};
 use file_system::{set_io_type, IoType};
 use futures::{compat::Future01CompatExt, FutureExt};
+use keys::enc_end_key;
```

Comment:
I have noticed rustc doesn't complain about unused imports in raftstore-v2. Does anyone know why? cc @BusyJay @sticnarf

Reply:
Because #![allow(unused)] is added in the lib.rs.

Reply:
Oh, apparently this:
https://github.com/tikv/tikv/blob/fe997db4db8a5a096f8a45c0db3eb3c2e5879262/components/raftstore-v2/src/lib.rs#L24